packages/theme/src/button/vars.css.ts

```diff
@@ -7,10 +7,10 @@ const defaultButtonVars = {
     height: '44px'
   },
   web: {
-    height: '28px'
+    height: '44px'
```

Comment:
temp change that makes all button have consistent size.
Without this, you would have app header buttons 28px x 44px or we could make them 28px x 28px but then they do not highlight the whole app header on hover.

Once we have this deployed we can do a design review and reconciliation.
It feels like the IA around breakpoints needs review and then we update the styles as needed.

Some of the current issues with the spec.
- At mobile/tablet, the drawer can be opened from two places (top-left+bottom-right buttons). 
- The top-left switches from an open to a close button, what happens with the bottom-right button
- The drawer at mobile/tablet is not full width but it should maximise the use of space
- it's possible to have two icons on the screen that look the same and both open drawers
- the bottom-right hand button is somewhat hidden and hard to see, you have to know it is there

At present, styles are consistently driven by theme, so it's a known starting point which we can iterate from.